packages/ts/lit-form/src/Field.ts

```diff
@@ -201,6 +205,17 @@ export class VaadinFieldStrategy<T = any, E extends FieldElement<T> = FieldEleme
       this.element.invalid = this.#invalid;
     }
   }
+
+  override elementValidity(): boolean {
+    // Ignore the `invalid` property of the Vaadin component to avoid
+    // reading the component's internal old validation state and validate
+    // the element based on the current state.
+    const isElementInvalid = this.element.invalid;
+    this.element.invalid = false;
+    const valid = this.element.checkValidity?.() ?? true;
+    this.element.invalid = isElementInvalid;
+    return valid;
+  }
```

Comment:
Would it be possible to override `checkValidity` here to avoid introducing public API method? 

```suggestion
  override checkValidity(): boolean {
    // Ignore the `invalid` property of the Vaadin component to avoid
    // reading the component's internal old validation state and validate
    // the element based on the current state.
    const isElementInvalid = this.element.invalid;
    this.element.invalid = false;
    const valid = super.checkValidity();
    this.element.invalid = isElementInvalid;
    return valid;
  }
```